src/App.tsx

```diff
@@ -1,24 +1,128 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorMessage } from './components/ErrorMesage/ErrorMesage';
+import { FormTodo } from './components/FormTodo/FormTodo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 6926;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [hasError, setHasError] = useState(false);
+  const [errorMesage, setErrorMessage] = useState('');
+  const [todoTitle, setTodoTitle] = useState('');
+  const [filteredTodos, setFilteredTodos] = useState<Todo[] | null>(null);
+
+  useEffect(() => {
+    const fetchTodos = async () => {
+      try {
+        const todosFromServer = await getTodos(USER_ID);
+
+        setTodos(todosFromServer);
+      } catch {
+        setHasError(true);
+        setErrorMessage('Unable to update a todo');
+      }
+    };
+
+    fetchTodos();
+  }, []);
+
+  const addNewTodo = (title: string) => {
+    if (!title.length) {
+      setErrorMessage("Title can't be empty");
+    }
+
+    const newTodo = {
+      id: 0,
+      userId: USER_ID,
+      completed: false,
+      title,
+    };
+
+    return addTodo(USER_ID, newTodo)
+      .then((todo: Todo[]) => {
+        setTodos((prevTodos) => {
+          return prevTodos?.concat(todo) || null;
+        });
+      })
+      .catch(() => {
+        setErrorMessage('Unable to add a todo!');
+      });
+  };
+
+  const removeTodo = (id: number) => {
+    return deleteTodo(id)
+      .then(() => {
+        const newTodosList = todos && todos.filter(todo => todo.id !== id);
+
+        setTodos(newTodosList);
+      })
+      .catch(() => {
+        setErrorMessage('Unable to delete a todo');
+      });
+  };
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
```

Comment:
```suggestion


  return (
```

you don't need this check. delete it